virus/RedMiteAssociatedHypovirus1-virTemplate.md

```diff
@@ -0,0 +1,117 @@
+
+
+### Virus Genome {Q3}
+
+```{r}
+library(ggplot2)
+library(gggenes)
+example_genes <- data.frame(
+  gene = c("polyprotein", "RNA_dep_RNAP" ),
+  start = c(39, 283 ), 
+  end = c(427, 676 ),
+  strand = c("+", "+"),
+  feature = c("ORF", "ORF")
+)
+ggplot(example_genes, aes(xmin = start, xmax = end, y = gene, fill = gene)) +
+  geom_gene_arrow() +
+  scale_fill_brewer(palette = "Set3") +
+  theme_minimal() +
+  xlab("Position") +
+  ylab("Gene")
+
+```
+Figure: This figure depicts the ORF for RNA dependent RNAP and polyprotein. Both the ORF have have partial CNS. The start and stop locations for the ORF were collected from ORFfinder from blast. The polyprotein ORF had the amino acid code of MEDRQNDQNDDVFSFFSPSFSAATPSTLFNRSAYSSSSSSGDDESQPSVDDSNKRIDYMIQFLDRRLSEDGNHDGIGDGNGSDSLPEFVGKCGESGIFKVPIRSAVHPNRPPSLDVRPHPLRETQIGRFLRTMTSTERQLWTGGEDGALRVWEFSELYGS. There are several motifs in the catalytic center that contain a number of conserved amino acid residues[Tirosh et al., 2018]. Viruses that hold this strand of polyproteins usually are known to be a positive-strand RNA eukaryotic virus[Tirosh et al., 2018]. A polyprotein consists of chains of smaller proteins[Crépin  et al., 2015]. This mecanism was developed to organize virus proteomes[Crépin  et al., 2015]. 
+
+The RNA_dep_RNAP ORF had the amino acid code of MAVNEIFFDRYPDSILEHQIQVRPFNALKTKNMRNLNPEDIDQLITISGMVIRTSQLIPEMQEAFFQCQVCAHTTRVEMDRGRIAEPSVCGRCHTTHSMALIHNRSLFSDKQMIKLQESPEDMPAGQTPHTVILFAHNDLVDKVQPGDRXNVTGIYRAVPIRVNPRXSXVKSVYKTHIDVIHYRKTDAKRLHGLDEXAEQKLFSXKRVELA. This ORF sequence is primarily found in rotavirus. It is hypothesized that the RdRP in rotavirus undergoes confrontational rearrangements during the synthesis of RNA due to the canonical RdRP motifs and a priming loop[Patton, n.d]. RdRPs play a crucial role in the synthesis of RNA in RNA viruses[Modrow et al., 2013]. 
+
+Used Chatgpt to help with syntax of coding gggenes. 
+
+### Other (bonus) sections
```

Comment:
Throughout the markdown file, please change the section headers so that they are descriptive of what's going on, not the assignment headers. You're writing a document to report something, this document is not an "assignment" as far as the readers are concerned.